sorc/chgres_cube.fd/write_data.F90

```diff
@@ -542,6 +542,9 @@ subroutine write_fv3_atm_bndy_data_netcdf(localpet)
      error = nf90_put_att(ncid, nf90_global, 'source', 'FV3GFS GAUSSIAN NETCDF FILE')
    elseif (trim(input_type) == "grib2") then
      error = nf90_put_att(ncid, nf90_global, 'source', 'FV3GFS GRIB2 FILE')
+!--- not defined right just to avoide the FV3 code change
+   elseif (trim(input_type) == "ifs_latlon_netcdf") then
+     error = nf90_put_att(ncid, nf90_global, 'source', 'FV3GFS GAUSSIAN NETCDF FILE' )
    endif
```

Comment:
Someone doing a dump of the file will not know it was created from IFS data. That might be confusing. How much work is it to update the FV3 code?

Reply:
@GeorgeGayno-NOAA, @LarissaReames-NOAA and I discussed these requirements a while ago, and they are outlined in [this](https://github.com/ufs-community/UFS_UTILS/issues/179) issue.  We believe it would require a fairly considerable and coordinated effort between a number developers across labs and potentially GFDL to migrate code from these FV3 pre-processing routines into UFS_UTILS.  The alternative is to introduce a new source string for IFS (similar to what was done for grib2 files) and make sure that correct field modifications/adjustments are being applied to that specific string.

Reply:
I would consider creating a new source string for IFS to avoid potential confusion.